WireAPI/Sources/WireAPI/HTTP Client/HTTPStatusCode.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+/// A list of HTTP status codes
+
+public enum HTTPStatusCode: Int {
+
+    // MARK: Success - 2xx
+
+    /// ok - 200
+
+    case ok = 200
```

Comment:
yes we have Swagger documentation (and legacy codebase as well) to make sure the success / failure codes we use are correct. For now this file is only filled with status codes we use for the refactored APIS but as the refactoring evolves we might add some other success/failure code, this will be done gradually I guess

Reply:
Cool. Perhaps as things evolve it would make sense for the `success` case to take a collection of status codes.